--- happytimer-2.2.1.org/Makefile.am	2011-10-02 10:58:26.000000000 +0900
+++ happytimer-2.2.1/Makefile.am	2011-10-02 13:42:38.414894167 +0900
@@ -2,17 +2,5 @@
 
 ACLOCAL_AMFLAGS = ${ACLOCAL_FLAGS}
 
-happytimerdocdir = ${prefix}/share/doc/happytimer
-happytimerdoc_DATA = \
-	README\
-	COPYING\
-	AUTHORS\
-	ChangeLog\
-	INSTALL\
-	NEWS
-
 EXTRA_DIST = \
 	autogen.sh
-
-uninstall-local:
-	-rm -r $(happytimerdocdir)
